data-raw/toy_data.R

```diff
@@ -1,10 +1,10 @@
 # styler: off
 toy_data <- tibble::tribble(
-  ~bank, ~amount, ~amount_of_disctinct_products, ~pctr_risk_category,     ~tilt_sec,
-    "A",       5,                             1,               "low",    "sector_a",
-    "A",       5,                             1,               "low",    "sector_b",
-    "B",       2,                             2,               "medium", "sector_a",
-    "B",       10,                            2,               "high",   "sector_b",
+  ~bank, ~amount, ~company_name, ~amount_of_distinct_products, ~pctr_risk_category,  ~tilt_sec,
+  "A",       5,       "Peter",                            2,               "low", "sector_a",
+  "A",       5,       "Peter",                            2,              "high", "sector_b",
+  "B",      10,       "Mario",                            1,            "medium", "sector_c",
+  "C",       8,       "Mario",                            1,               "low", "sector_c"
```

Comment:
This looks miss-aligned by 1-2 characters?

Reply:
Oops, you're right. I'll change that in a separate PR so that Anne can test the code with the Bundesbank :) 
Thank you for your comments @maurolepore. I think the next step would be to transfer into a ggplot instead of networkD3 package and DRY the code ! :)